.idea/copyright/GPLv3.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="notice" value="AsteroidOSSync &#10;Copyright (c) &amp;#36;today.year AsteroidOS&#10;&#10;This program is free software: you can redistribute it and/or modify&#10;it under the terms of the GNU General Public License as published by&#10;the Free Software Foundation, either version 3 of the License, or&#10;(at your option) any later version.&#10;&#10;This program is distributed in the hope that it will be useful,&#10;but WITHOUT ANY WARRANTY; without even the implied warranty of&#10;MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the&#10;GNU General Public License for more details.&#10;&#10;You should have received a copy of the GNU General Public License&#10;along with this program. If not, see &lt;http://www.gnu.org/licenses/&gt;." />
+    <option name="myName" value="GPLv3" />
+  </copyright>
+</component>
```

Comment:
This line doesn't end with a newline character.
A valid line is defined as `A sequence of zero or more non- <newline>s plus a terminating <newline>`.

https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap03.html#tag_03_205

Reply:
Intelij gets mad if you add a newline in this.